src/content/docs/en/recipes/i18n.mdx

```diff
@@ -7,9 +7,8 @@ i18nReady: true
 import FileTree from '~/components/FileTree.astro'
 import StaticSsrTabs from '~/components/tabs/StaticSsrTabs.astro'
 
-Astro doesn't have built-in internationalization (i18n) support, but you can build your own i18n solution.
+In v4.0, Astro received [built-in support for internationalization (i18n) routing](/en/guides/internationalization/). However, if you prefer, you can build your own i18n solution.
 In this recipe, you'll learn how to use content collections and dynamic routing to serve content for different languages.
```

Comment:
```suggestion
In this recipe, you'll learn how to use content collections and dynamic routing to build your own solution to serve content for different languages.

:::tip
In v4.0, Astro added built-in support for internationalization (i18n) routing that allows you to configure default and supported languages and includes useful helper functions to assist you in serving an international audience. Consult our [internationalization guide](/en/guides/internationalization/) to learn about these features.
:::
```

The hardest part... remembering that you can blow up what's in front of you, and try something else entirely!  😄 

What about something like this?

Reply:
I like it! I will keep that in mind for future docs changes such as this. I will read this, and see if I can improve. If not, it works for me! I will keep you updated :)

Reply:
I will wait for your final attempt/verdict!  🫡

Reply:
```suggestion
In this recipe, you will learn how to use content collections and dynamic routing to build your own i18n solution to serve content in different languages.

:::tip
In v4.0, Astro added built-in support for internationalization (i18n) routing that allows you to configure default and supported languages and includes valuable helper functions to assist you in serving an international audience. If you want to use this instead, see our [internationalization guide](/en/guides/internationalization/) to learn about these features.
:::
```

@sarah11918 How is this? If it's good, I'll commit it and we are good to merge whenever you are ready!

Reply:
It has been absolutely, unequivocally shipped! 🚀 thanks for the help :)